docs/authentication/configuration/sign-up-sign-in-options.mdx

```diff
@@ -69,7 +60,7 @@ To configure authentication strategies:
 
 A passkey is a type of sign-in credential that requires one user action, but uses two authentication factors:
   1. A pin number or biometric data
-  1. A physical device
+  2. A physical device
```

Comment:
Markdown auto-numbers ordered lists, so it's better to just use 1 for each item. That way if you ever need to swap their positions, you don't have to change the number. Easier to maintain. Here's how the list looks in prod:

![image](https://github.com/clerk/clerk-docs/assets/5156915/eca7ec42-c3c0-4008-a332-f2805676d617)